eva/catalog/sql_config.py

```diff
@@ -58,10 +48,20 @@ def __init__(self):
         Retrieves the database uri for connection from ConfigurationManager.
         """
         uri = ConfigurationManager().get_value("core", "catalog_database_uri")
+
         # parallelize using xdist
-        # worker_uri = prefix_worker_id(str(uri))
+        def prefix_worker_id(uri: str):
```

Comment:
How is it different from the similar function in `utils`?

Reply:
One of them is for a database uri. The other one is for a directory path.